channels/am.m3u

```diff
@@ -1,9 +1,9 @@
 #EXTM3U
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Armenian" tvg-logo="https://i.imgur.com/3vQVwUV.jpg" group-title="",Amga
 https://streamer1.connectto.com/AMGA_WEB_1202/tracks-v1a1/mono.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Armenian" tvg-logo="https://i.imgur.com/MKVkDVo.png" group-title="",H1
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Armenian" tvg-logo="https://i.imgur.com/MKVkDVo.png" group-title="",H1 (Audio Track 1)
```

Comment:
The channel names should only be the channel.

Reply:
Thing is there's something linked to the audio tracks in the stream. Shall I delete the mention and count the second one as a backup ?

Reply:
that works.